docs/src/main/asciidoc/qute.adoc

```diff
@@ -162,6 +162,13 @@ Hello {name}! <1>
 ----
 <1> `{name}` is a value expression that is evaluated when the template is rendered.
 
+.HelloResource/goodebye.txt
+[source]
+----
+Goodbye {name}! <1>
+----
+<1> `{name}` is a value expression that is evaluated when the template is rendered.
+
```

Comment:
These lines should not be added, imho.
But it is important to rename [line 211](https://github.com/quarkusio/quarkus/blob/de0a859622edba0830e2427ba182f2cf2103d2f5/docs/src/main/asciidoc/qute.adoc?plain=1#L211) from `HelloResource.java` to `GoodbyeResource.java`: https://github.com/quarkusio/quarkus/blob/de0a859622edba0830e2427ba182f2cf2103d2f5/docs/src/main/asciidoc/qute.adoc?plain=1#L211-L226

Reply:
Also, add
```java
public static native TemplateInstance goodbye(String name);
```
after https://github.com/quarkusio/quarkus/blob/de0a859622edba0830e2427ba182f2cf2103d2f5/docs/src/main/asciidoc/qute.adoc?plain=1#L194
<br>
And change https://github.com/quarkusio/quarkus/blob/de0a859622edba0830e2427ba182f2cf2103d2f5/docs/src/main/asciidoc/qute.adoc?plain=1#L231 in GoodbyeResource.java to
```java
return HelloResource.Templates.goodbye(name);
```

Reply:
> These lines should not be added, imho.

Sorry, I was confused. They should stay of course.
But there's a typo in goodbye.txt: https://github.com/quarkusio/quarkus/blob/f32455e7570b08a41ace95fafa3f9223e6404104/docs/src/main/asciidoc/qute.adoc?plain=1#L165

And https://github.com/quarkusio/quarkus/blob/f32455e7570b08a41ace95fafa3f9223e6404104/docs/src/main/asciidoc/qute.adoc?plain=1#L156 should be changed to
```md
We'll start with two very simple templates:
```